github/provider_test.go

```diff
@@ -90,6 +102,59 @@ func TestProvider_individual(t *testing.T) {
 				Config:      individualProviderConfig + testAccGithubMembershipConfig(username),
 				ExpectError: regexp.MustCompile("This resource requires GitHub organization to be set on the provider."),
 			},
+			{
+				Config:      conflictingProviderConfig + testAccCheckGithubUserDataSourceConfig(username),
+				ExpectError: regexp.MustCompile("If `individual` is true, `organization` cannot be set."),
```

Comment:
Does this test pass? `conflictingProviderConfig`, from what I can see, only sets `individual` to `true`, it doesn't set an organization.